src/__tests__/common/helper.ts

```diff
@@ -39,3 +39,16 @@ export const getErrorMessage = (isQuery = false): HTMLElement => {
 	}
 	return screen.getByText(ERROR_MESSAGE);
 };
+
```

Comment:
`getResetButtonProps` is not used? can remove it if that is the case.

I also don't see `getResetButton()` used anywhere else other than the reset-button tests, can move it to the reset-button tests first